packages/protocol/scripts/foundry/create_and_migrate_anvil_devchain.sh

```diff
@@ -60,6 +60,25 @@ $NON_INTERACTIVE \
 $LIBRARY_FLAGS \
 --rpc-url $ANVIL_RPC_URL || echo "Migration script failed"
 
+# Set the supply for Celo:
+# This supply is just an approximation, but it's importat that this number is non-zero.
+
+# get celo contract address
+CELO_ADDRESS=`cast call $REGISTRY_ADDRESS "getAddressForStringOrDie(string calldata identifier) returns (address)" "CeloToken" --rpc-url $ANVIL_RPC_URL`
+
+# interpersonate the VM, this can't be done from the migration script
+cast rpc anvil_impersonateAccount $CELO_VM_ADDRESS --rpc-url $ANVIL_RPC_URL
+
+# set the balance of the vm address so that it can send a tx
+cast rpc anvil_setBalance $CELO_VM_ADDRESS 1000000000000000000 --rpc-url $ANVIL_RPC_URL
+
+# increase the supply
+# 540001000000000000000000 = (60e3 + 1) * 9 * 10^18
```

Comment:
Could you leave a note on what the numbers are. It's great there is a formula, but I'm not sure I would understand where the numbers come from.

Feel free to define them in [`constants.sh`](https://github.com/celo-org/celo-monorepo/blob/release/core-contracts/12/packages/protocol/scripts/foundry/constants.sh) and leave verbose comments there.

Reply:
This seems to be done